index.js

```diff
@@ -0,0 +1,18 @@
+function receivesAFunction(cb) {
+  return cb();
+}
+
+receivesAFunction(function () {
+  return "What is love?";
+});
+
+function returnsANamedFunction() {
+  function noMore() {
+    return "Baby don't hurt me.";
+  }
+  return noMore;
```

Comment:
You can even just do this:
```suggestion
  return function noMore() {
    return "Baby don't hurt me.";
  }
```